features/directory_structure.feature

```diff
@@ -67,6 +67,19 @@ Feature: Directory Structure
   configured `infer_spec_type_from_file_location!`, RSpec will automatically
   include the correct support functions for each type.
 
+  **Note:** `infer_spec_type_from_file_location!` affects only **the fixed list of folders**: controllers, helpers, jobs, mailers, models, requests, integration, api, routing, views, features.
```

Comment:
I see this line as rather needless as the line directly above it says that: "If you follow the above listed canonical directory structure..." And the canonical structure mentioned is the very first thing this document goes over in detail.

Perhaps we can remove this line and just change the one below to "If you want to set metadata for a custom directory that doesn't follow fit the canonical structure above, you can do the following:" ? (Note I changed folder -> directory as well, since the rest of the document uses directory, not folder. Consistency FTW!)